CDCgov/prime-reportstream
# 8598 update CA Jurisdictional filter

This PR shows what settings changes are needed to update the jurisdictional filter for the California DPH. The original filter would route results to California if the testing company happened to be based in California even if the patient was located in another state and was tested there.

Test Steps:
1. Load these settings into your local repo (removing the transport section)
2. Send test data where the `ordering_facility_state` is `CA` but the `ordering_provider_state` is not.
3. Ensure data is not routed to CA

## Changes
- jurisdictional filter for CA-DPH

### Testing

### Process

## Linked Issues
- 8598

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
